src/components/LearningCard.js

```diff
@@ -0,0 +1,88 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+import ButtonExpansion from './ButtonExpansion'
+import { useState } from 'react'
+import GlobalStyle from '../GlobalStyle'
+
+Card.propTypes = {
+  onClick: PropTypes.func,
+  isExpanded: PropTypes.bool,
+  name: PropTypes.string,
+  description: PropTypes.string,
+  tags: PropTypes.array,
+  color: PropTypes.string,
+}
+
+export default function Card({ name, description, tags, color }) {
+  const [isExpanded, setIsExpanded] = useState(false)
+
+  return (
+    <>
+      <GlobalStyle />
+      <LearningCard isExpanded={isExpanded} backgroundColor={color}>
+        <h2 isExpanded={isExpanded}>{name}</h2>
```

Comment:
This doesn't do anything here as a normal `h2` is not a styled-component. The reason it works is because of giving the props to `LearningCard`.
```suggestion
        <h2>{name}</h2>
```